packages/asc-ui/src/components/ShowMoreShowLess/ShowMoreShowLessStyle.ts

```diff
@@ -23,19 +23,14 @@ export default styled.div<StyleProps>`
 
       ${showMoreShowLess &&
       css`
-        &:after {
-          content: '';
-          display: block;
-          width: 100%;
-          height: 100px;
-          background: linear-gradient(
-            180deg,
-            rgba(255, 255, 255, 0) 0%,
-            rgba(255, 255, 255, 1) 100%
-          );
-          position: absolute;
-          bottom: 0;
-        }
+        mask-image: -webkit-gradient(
```

Comment:
Autoprefixer doesn't take care of this for us? Can't we add a version that is standards compatible?